ansible/roles/system/tasks/main.yml

```diff
@@ -1,33 +1,37 @@
-
-#- name: test for available disk space
-#  assert:
-#    that:
-#     - "{{ item.size_available > 500 * 1000 * 1000 }}" # 500Mb
-#  when: "{{ item.mount == '/' }}"
-#  with_items: "{{ ansible_mounts }}"
+---
```

Comment:
Remove this block altogether. We can always revert it back from git history if needed.